src/lib.rs

```diff
@@ -32,33 +32,51 @@ use log::Level;
 pub struct Verbosity {
     /// Pass many times for more log output
     ///
-    /// By default, it'll only report errors. Passing `-v` one time also prints
-    /// warnings, `-vv` enables info logging, `-vvv` debug, and `-vvvv` trace.
+    /// By default it'll report log level info. Passing `-v` one time also
+    /// prints debug, `-vv` enables trace logging.
     #[structopt(long = "verbosity", short = "v", parse(from_occurrences))]
     verbosity: u8,
+    /// Suppress all log output.
+    ///
+    /// If you also pass `-v` it'll report errors. Passing `-vv` will also print
+    /// warnings, `-vvv` enables info logging, `-vvvv` debug, and `-vvvvv`
+    /// trace.
+    #[structopt(long = "quiet", short = "q")]
+    quiet: bool,
```

Comment:
Another option is to allow repeated --quiet.  It might not be obvious that to be completely quiet you need `-qq`.

Reply:
Oh yeah, that's a cool suggestion! If that were the case we should also make the flags exclusive by putting them in a group. I don't know which one is better though. What do you think?

Reply:
I can go either way.  Both can have confusing situations and there is consensus on q/v existing but not how to handle multiple logging levels.

btw here is my python code I've been using for 5-10 years
```python
# We want to default to WARNING
# Verbosity gives us granularity to control past that
if 0 < args.verbose and 0 < args.quiet:
    parser.error("Mixing --verbose and --quiet is contradictory")
verbosity = 2 + args.quiet - args.verbose
verbosity = max(verbosity, 0)
verbosity = min(verbosity, 4)
args.logging_level = {
    0: logging.DEBUG,
    1: logging.INFO,
    2: logging.WARNING,
    3: logging.ERROR,
    4: logging.CRITICAL,
}[verbosity]
```